pyproject.toml

```diff
@@ -29,12 +29,13 @@ documentation = "https://docs.camel-ai.org"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<3.12"
 numpy = "^1"
-openai = "^1.2.3"
+openai = ">=1.2.3,<1.14.0"
 tiktoken = "^0"
 colorama = "^0"
 jsonschema = "^4"
 protobuf = "^4"
 pathlib = "^1.0.1"   
+anthropic = "^0.7.7"
```

Comment:
move it into [tool.poetry.extras] part instead keep it in [tool.poetry.dependencies] to reduce default installation?

Reply:
I don't agree since we have a whole file describes how to treat anthropic models and it need anthropic package itself. Move the requirement as extra will simply break it. It is as import as the `openai` package
https://github.com/camel-ai/camel/pull/288/files#diff-f90260a7e58e0c09b9d2e737a580635d71a7310f1a4adebdb3b41323a960698cR17